Marlin/src/HAL/HAL_STM32F1/HAL_spi_STM32F1.cpp

```diff
@@ -79,15 +79,25 @@ void spiBegin() {
  * @details
  */
 void spiInit(uint8_t spiRate) {
+  /**
+   * STM32F1 APB1 = 72MHz, APB2 = 36MHz, max SPI speed of this MCU if 18Mhz 
```

Comment:
comment APB1/2 is reversed, cf the capture of the datasheet

Reply:
So, this?
```cpp
* STM32F1 APB1 = 36MHz, APB2 = 72MHz, max SPI speed of this MCU if 18Mhz
```

Reply:
I see below, yes.

Reply:
yes APB2 is named high speed periph bus... APB1 low speed

Reply:
Oh, just found that : https://github.com/rogerclarkmelbourne/Arduino_STM32/blob/master/STM32F1/libraries/SPI/src/SPI.cpp#L776

Reply:
Hmm… Yes, `determine_baud_rate` could be kind of useful.